### ISSUE TITLE
Preview hangs when editing mj-attributes

### ISSUE THREAD

**Issue by daGUY** [attilabuti/vscode-mjml#45]
I often use the `<mj-attributes>` tag to set common styles for things like text size & color, buttons, etc.

If I have the MJML preview pane open, it works fine and updates live as long as I'm editing within the `<mj-body>` tag. But as soon as I edit anything inside of the `<mj-attributes>` tag, the preview stops updating and my CPU usage spikes to 100%. The only way to get it to work again is to quit and relaunch VS Code (if I simply close the preview pane without quitting VS Code, it won't reopen again).

I know that the code changes I'm making within the `<mj-attributes>` tag are valid MJML, because they'll appear in the preview after I quit and relaunch VS Code. So it's not tripping up on invalid syntax or something like that.

I'm running:

- VS Code 1.28.2
- vscode-mjml 1.5.1
- macOS 10.14

This happened to me on earlier versions of VS Code and macOS as well, but I didn't figure out that it was the `<mj-attributes>` in particular that was causing the issue until now.

**Comment by attilabuti** [comment]
Hello @daGUY,

Can you send me a snippet of code that reproduces the error?

**Comment by daGUY** [comment]
Just any edits inside of the `<mj-attributes>` tag seem to trigger it. Here's what I have on the particular project I'm working on now:

```
<mjml>
    <mj-head>
        <mj-attributes>
            <mj-button align="left" background-color="transparent" color="#007aff" font-family="Arial, Helvetica, sans-serif" font-weight="bold" inner-padding="0" />
            <mj-text font-family="Arial, Helvetica, sans-serif" />
        </mj-attributes>
    </mj-head>
    <mj-body>
        ...
    </mj-body>
</mjml>

So for example, if I change the `align="left"` to `align="center"`, then the preview will stop working. Then if I quit and relaunch VS Code, the preview will show the change to `align="center"` but will again stop working if I make any other changes. The preview works fine as long as any changes I make are inside of `<mj-body>`, but as soon as I edit anything inside `<mj-attributes>` then it breaks until I quit and relaunch.

**Comment by attilabuti** [comment]
Hello @daGUY,

The issue is probably fixed on 1.6.0.

**Comment by daGUY** [comment]
@attilabuti , so I actually came here to mention that I haven't had any issues like this for the past few weeks, not realizing that a new version of the plugin was released (since VS Code plugins auto-update in the background). I checked just now and indeed I'm running 1.6.0, so that seems to have done the trick. Thanks!
